massa-versioning/src/lib.rs

```diff
@@ -9,13 +9,22 @@
 //! MIPState -> Deployment state of a MIPInfo
 //! MIPStore -> A map of MIPInfo -> MipState
 //!
-//! # Note on MipInfo versions
+//! # Notes on MipInfo versions
 //!
-//! There is 2 different versions:
+//! There is 2 different 'versions':
 //! * version == Network version -> This is the network version to announce and thus to store in block header
-//! * component_version -> This is the version for the associated component and is used in VersioningFactory
+//! * component_version -> This is the version for the associated component and is used in VersioningFactory (e.g KeyPair, Address, VM)
 //!
-//! # Note on MipState
+//! # Notes on MipInfo timings and stats
+//!
+//! So in the execution module and after a block become final, we update the MipStore stats (in a blocking way) then continue one.
+//! By updating the stats, we mean sending: (Slot timestamp, Option<(current: u32, advertising: Option)>).
+//! Using the slot timestamp, ensure that the trigger (and the trigger timeout) is a consensus by all nodes
+//! (This means that the trigger and the trigger timeout are not timer based).
+//! In order to have all nodes in sync (because of node various delays), the state is se to active
+//! after an activation delay (duration is required to be > 1 cycle).
```

Comment:
I there supposed to be a check on duration being above 1 cycle? Used activation delays lower than that in testing

Reply:
For testing purpose, there is no limit for now. I'm adding this as a followup here: https://github.com/massalabs/massa/issues/3631